readthedocs/projects/models.py

```diff
@@ -485,6 +485,14 @@ def single_version_symlink_path(self):
     #
 
     def venv_bin(self, version=LATEST, bin='python'):
```

Comment:
Might make sense to just kill the python default here, so that we don't have to pass None everywhere